CorsaSchoolManagementSystem/Views/dashboard.php

```diff
@@ -220,7 +220,32 @@
                             </div>
                         </div>
                     </div>
-                </div>
+                </div> -->
+
+                <!-- Class view -->
+                <div class="container search mt-5">
+                    <table id="example1" class="table table-striped table-bordered nowrap" style="width:100%">
+                        <thead>
+                            <tr>
+                                <th>Class Name</th>
+                                <th>Subjects</th>
+                                <th>Class Teacher</th>
+                                <th>Number Of Students</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <?php foreach($addClass as $classData): ?>
+                               <tr>
+                               <td><?php echo $classData->name ?></td>
+                                <td><?php echo $classData->subjects ?></td>
```

Comment:
Instead of showing subjects as json array in the view, can you find out how to split the json array into list of subjects separated by commas.